boards/lilygo/ttgo_lora32/doc/index.rst

```diff
@@ -220,6 +220,26 @@ To build the LoRa receive sample application using sysbuild use the command:
    :west-args: --sysbuild
    :compact:
 
+The following sample application lists the contents of the SD card:
+
+.. zephyr-app-commands::
+   :tool: west
+   :zephyr-app: samples/subsys/fs/fs_sample
+   :board: ttgo_lora32/esp32/procpu
+   :goals: build
+   :west-args: --sysbuild
+   :compact:
+
+The following sample application displays numbers on the OLED display:
+
+.. zephyr-app-commands::
+   :tool: west
+   :zephyr-app: samples/subsys/display/cfb
+   :board: ttgo_lora32/esp32/procpu
+   :goals: build
+   :west-args: --sysbuild
+   :compact:
```

Comment:
Not blocking but I'd rather you link to the samples instead. I understand that this is useful to give ready to copy-paste build instructions but it's also content that might be left behind should something change in the way these samples should be built (arguably unlikely, but still)
I would simply say something like "You may also run :zephyr:code-sample:`XXX` to blah blah or ... to ...".

Reply:
Hi @kartben 
Yes, I can change them to be links.
Since I usually compare with documentation files from other boards, is there a reference documentation you'd recommend? Especially how to make the links.
The copy-paste build instructions did actually change not so long ago, and I had to update them. So, removing those would be good.

Reply:
Should I add links like this?

* :zephyr:code-sample:`lora-send`
* :zephyr:code-sample:`lora-receive`
* :zephyr:code-sample:`fs-fs-sample`
* :zephyr:code-sample:`display-cfb`

(how do subdirectories/catgories get handled?)